src/main/java/com/kustudents/issuetracker/controller/viwIssuesController.java

```diff
@@ -0,0 +1,30 @@
+package com.kustudents.issuetracker.controller;
+
+
+import java.util.List;
+
+import org.springframework.web.bind.annotation.*;
+import com.kustudents.issuetracker.data.enitity.viwIssues;
+import com.kustudents.issuetracker.service.viwIssuesService;
+
+@RestController
+@RequestMapping("/api/Issues")
```

Comment:
I think, there is a way to not include "/api/" in the mapping (at least we didn't have to in my last project). I will look into that